eksperymenty_programistyczne_na_edoku4.rst

```diff
@@ -0,0 +1,91 @@
+Eksperymenty programistyczne na eDoku4
+======================================
+
+Pierwszy eksperyment programistyczny
+------------------------------------
+
+Cel eksperymentu
+~~~~~~~~~~~~~~~~
+
+Celem eksperymentu jest wstępne wysondowanie możliwości rozbudowy testowanego systemu. Jego wynik może pomóc w opracowaniu kolejnych, bardziej wyszukanych eksperymentów.
+
+Założenia eksperymentu
+~~~~~~~~~~~~~~~~~~~~~~
+
+Instrukcja kancelaryjna w ramach minimalnego zestawu metadanych opisujących przesyłki wpływające oraz wychodzące wymienia (opcjonalny) adres email odpowiednio nadawcy oraz adresata. Urząd mógłby chcieć wdrożyć narzędzie umożliwiające Obywatelom zalogowanie się swoim adresem email oraz wyświetlenie przesyłek powiązanych ze swoim kontem. Eksperyment ma za zadanie ustalić możliwość łatwego napisania takiej aplikacji.
```

Comment:
Mam poważne wątpliwości, pomijając przypuszczalnie szczytne intencje, czy poprawne językowo jest w takim kontekście pisanie słowa "Obywatelom" z wielkiej litery. Dostrzegam jednak, że może to być jednak element polityki organizacji.